mlflow/catboost.py

```diff
@@ -328,7 +329,16 @@ class _CatboostModelWrapper:
     def __init__(self, cb_model):
         self.cb_model = cb_model
 
-    def predict(self, dataframe):
+    def predict(self, dataframe, params: Optional[Dict[str, Any]] = None):
```

Comment:
I think pylint will complain this argument is unused.

Reply:
Ah okay